winml/lib/Api/HardwareCoreEnumerator.cpp

```diff
@@ -14,7 +14,7 @@ struct LogicalProcessorInformation {
 
```

Comment:
Please split this change to a separate PR. The WinML team will review it.

Reply:
@smk2007 can you review this PR from WinML perspective?